src/components/plots/piePlot.ts

```diff
@@ -2,6 +2,12 @@
 
 module Plottable {
 export module Plots {
+
+  export interface ValueScaleBinding<V, D, R> {
+    value: V | _Accessor | D;
```

Comment:
For `innerRadius` and `outerRadius` you want your `value`s to be `number`s or `Accessor`s to a number.  Thus, how would you encapsulate that into a type?